src/Engine/GUI/Workspaces/OrbitalWorkspace.cpp

```
@@ -185,13 +195,56 @@ void OrbitalWorkspace::loadWorkspaceConfig(const std::string &configPath) {
+			FilePathUtils::JoinPaths(ROOT_DIR, simulationDir, m_simulationName + ".yaml").c_str(),
+			SIZE_OF(filters),
+			filters,
+			NULL
+		);
+		if (!selected || selected == "0" || strlen(selected) == 0)
+			return false;
+
+
+		std::string selectedFilePath(selected);
+
+		if (selectedFilePath.empty())
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)